static/js/message_list_view.js

```diff
@@ -34,6 +34,8 @@ import * as submessage from "./submessage";
 import * as timerender from "./timerender";
 import * as util from "./util";
 
+const RESOLVED_TOPIC_PREFIX = "✔ ";
```

Comment:
Let's do a preparatory refactoring commit moving this to be exported as `message_edit.RESOLVED_TOPIC_PREFIX`, so that we don't end up with multiple copies of this definition.